frontends/web/src/components/headerssync/headerssync.tsx

```diff
@@ -16,89 +16,48 @@
  * limitations under the License.
  */
 
-import { Component} from 'react';
+import { FunctionComponent } from 'react';
+import { useTranslation } from 'react-i18next';
 import { CoinCode } from '../../api/account';
-import { subscribe } from '../../decorators/subscribe';
-import { translate, TranslateProps } from '../../decorators/translate';
+import { subscribeCoinHeaders } from '../../api/coins';
+import { useSubscribe } from '../../hooks/api';
 import Spinner from '../spinner/ascii';
 import style from './headerssync.module.css';
 
-interface IHeadersSyncProps {
+interface Props {
     coinCode: CoinCode;
 }
 
-interface IStatus {
-    targetHeight: number;
-    tip: number;
-    tipAtInitTime: number;
-    tipHashHex: string;
-}
-
-interface ISubscribedHeadersSyncProps {
-    status?: IStatus;
-}
-
-interface IState {
-    show: number;
-}
-
-type Props = ISubscribedHeadersSyncProps & IHeadersSyncProps & TranslateProps;
+export const HeadersSync: FunctionComponent<Props> = ({coinCode}) => {
+    const { t } = useTranslation();
+    const status = useSubscribe(subscribeCoinHeaders(coinCode));
 
-class HeadersSync extends Component<Props, IState> {
-    public readonly state: IState = {
-        show: 0,
+    if (!status) {
+        return null;
     }
 
-    componentDidUpdate(prevProps) {
-        const { status } = this.props;
-        if (status && prevProps.status && status.tip !== prevProps.status.tip) {
-            this.setState({ show: status.tip });
-            if (status.tip === status.targetHeight) {
-                // hide component after 4s when tip reached targetHeight
```

Comment:
the commit msg does not explain why `show` is not needed. what about this comment? did this functionality change?

Reply:
I added an eplanation about why i've removed it:

- removed unnescesary hiding logic on `HeadersSync` `HeadersSync` was returning `null` after 4 seconds the headers been sync, this was a display implementation based on its usage, meanwhile the only usage it's on `account.tsx`, where the component is removed from render just after the headers been sync, 
being unnescesary the hidding of the component, alongside it's a feature that should be in `account.tsx` if wanted, not in the component

Reply:
Somehow it didn't remove the comp after synced, not sure why.

![Screen Shot 2021-12-21 at 5 23 30 PM](https://user-images.githubusercontent.com/546900/146964354-c0702867-1c5c-433e-8c5d-6e842e36d0c5.png)

Reply:
I've readded the hidding logic